src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,9 +17,12 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
-
+    public static final String USERGUIDE_URL =
+            "https://github.com/AY2021S1-CS2103T-T13-1/tp/blob/master/docs/UserGuide.md";
```

Comment:
For the user guide link, it doesn't look user friendly to give the github link. Please use the formal website link instead as follows. The format is similar to the old addressbook user guide link.
https://ay2021s1-cs2103t-t13-1.github.io/tp/UserGuide.html

Reply:
ohh thanks, was looking for this link